en/identity-server/next/docs/apis/restapis/actions.yaml

```diff
@@ -0,0 +1,753 @@
+openapi: 3.0.0
+info:
+  version: v1
+  title: WSO2 Identity Server - Action Management Rest API
+  description:
+    "This is the RESTful API for managing action configuration for an organization in WSO2 Identity Server. \n
+    This API allows creating, retrieving, updating and deleting the action configurations."
+servers:
+  - url: 'https://{server-url}/t/{tenant-domain}/api/server/v1'
+    variables:
+      server-url:
+        default: "localhost:9443"
+      tenant-domain:
+        default: carbon.super
+security:
+  - BasicAuth: []
+  - OAuth2: []
+paths:
+  /actions/types:
+    get:
+      tags:
+      - Actions Endpoint
+      summary: List Action Types
+      description: "This API returns the detailed summary of the action types.\n\n
+      - <b>Scope(Permission) required:</b> ``internal_action_mgt_view``\n\n"
+      operationId: getActionTypes
+      responses:
+        '200':
+          description: Action Types are found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ActionTypesResponse'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
```

Comment:
example curls are missing for each endpoint

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4700/commits/7e686655a3f252d7ee7c7fe8c4d391177b53733d